yaml/buildings.yaml

```diff
@@ -62,9 +62,13 @@ globals:
         way_area: { min: 500 }
         volume: { min: 50000 }
   - &z15_area_volume
-      way_area: { min: 20 }
+      any:
+        way_area: { min: 50 }
```

Comment:
These are working well in Barcelona and NYC. Possibly we could be more aggressive, but seems fine.